apps/api/src/app/workflows-v2/workflow.controller.e2e.ts

```diff
@@ -243,6 +245,7 @@ function buildCreateWorkflowDto(nameSuffix: string): CreateWorkflowDto {
   return {
     __source: WorkflowCreationSourceEnum.EDITOR,
     name: TEST_WORKFLOW_NAME + nameSuffix,
+    workflowId: `${slugifyName(TEST_WORKFLOW_NAME + nameSuffix)}`,
```

Comment:
I think it's an antipattern to supply a slugified `workflowId` during workflow creation, that concern should be handled in the API. Am I missing something?

Reply:
Good point i made it like this because of the following design where the client has validation.

![image](https://github.com/user-attachments/assets/f717b6a3-bc13-4359-8e6b-f8646c0e5cad)